Makefile

```diff
@@ -52,3 +68,6 @@ publish-%: ## publish a new version from master. Argument will be fed into `npm
 	@git diff --quiet || (echo 'unclean git repo, please commit all changes before publish'; exit 1)
 	export VERSION=$$(npm version patch) && git checkout -b dahlia/$$VERSION && npm publish --access public && git push --set-upstream origin dahlia/$$VERSION && git push --tags
 	echo "NPM package published, git branch created and tags pushed. Go to https://github.com/edx/studio-frontend to see your PR and merge the package.json update"
+
+validate-devstack-folders:
+	docker exec -t edx.devstack.studio bash -c 'if [[ $$(ls /edx/src/sfe_parent/symlinks) && $$(ls /edx/src/sfe_parent/studio-frontend) ]]; then echo "Folder structure looks good"; else echo "Your folders are FUBAR, talk to Eric (who should really come up with an automatic way of repairing this situation)"; exit 1; fi'
```

Comment:
This verifies that the folders exist on the devstack containers. Is that good enough, or should we add in tougher checks like "the reported git status of the container folder matches the folder we're currently executing from"?